packages/coinjoin/src/client/Account.ts

```diff
@@ -1,11 +1,28 @@
 import { Network } from '@trezor/utxo-lib';
 
-import { getScriptPubKeyFromAddress } from '../utils/coordinatorUtils';
+import { getScriptPubKeyFromAddress, prefixScriptPubKey } from '../utils/coordinatorUtils';
 import { AllowedScriptTypes } from '../types/coordinator';
 import { RawLiquidityClue } from '../types/middleware';
 import { RegisterAccountParams } from '../types';
 import { AccountUtxo, AccountAddress } from '../types/account';
 
+const enhanceAccountUtxo = (
+    utxos: Omit<AccountUtxo, 'scriptPubKey'>[],
+    network: Network,
+    scriptType: AllowedScriptTypes,
+): AccountUtxo[] =>
+    utxos.flatMap(address => ({
+        path: address.path,
+        address: address.address,
+        scriptPubKey: prefixScriptPubKey(
+            getScriptPubKeyFromAddress(address.address, network, scriptType),
+            true,
+        ),
+        outpoint: address.outpoint,
+        amount: address.amount,
+        anonymityLevel: address.anonymityLevel,
+    }));
```

Comment:
```suggestion
    utxos.flatMap(address => ({
        ...address,
       scriptPubKey: prefixScriptPubKey(
            getScriptPubKeyFromAddress(address.address, network, scriptType),
            true,
        ),
    }));
```

Reply:
i'm not sure if it's safe approach to spread incoming data objects in any api-like library

Reply:
better not to spread here